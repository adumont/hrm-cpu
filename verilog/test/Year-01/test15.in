12 d6
39 dd
06 fb
30 80
42 e0
