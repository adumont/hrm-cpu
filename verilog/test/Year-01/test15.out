d6
dd
fb
